### DISCUSSION TITLE
Show queued (to be run) jobs?

### DISCUSSION THREAD

**Discussion by leenooks** [romanzipp/Laravel-Queue-Monitor#62] [Ideas]
Howdy, just came across your tool and it will help me with something that I'm trying to do - thank you.

Have you considered showing yet to be run jobs - ie: jobs that have been queued?

I think that would be really useful in this tool...

**Comment by kaffeeguru** [comment]
I would be also interested in this feature, did you try it before and are there any known obstacles on the way to reach this?

**Comment by bessone** [comment]
Hello,

I have the same question, I only see the jobs in the queue monitor when they start running, is it not possible to see all the ones in the queue before they are executed so that I can know how many still have to be processed?
